app/src/main/kotlin/com/hedvig/app/feature/claims/ui/ClaimsFlowActivity.kt

```diff
@@ -0,0 +1,40 @@
+package com.hedvig.app.feature.claims.ui
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
+import com.hedvig.app.authenticate.AuthenticationTokenService
+import com.hedvig.common.nav.OdysseyNavigationComposable
+import org.koin.android.ext.android.inject
+
+class ClaimsFlowActivity : ComponentActivity() {
```

Comment:
Is this the way we're going to be working with different flows?
Will it not be more like OdysseyActivity, and then the inner router will take care of any and all navigation within it? Or maybe something else?

Reply:
The way the library API looks like now, we take a `navigationRootUrl` which specifies which router Odyssey will show as "root". For now we only have one flow, claims recording, so this URL can be hard coded in this activity. 

Later on we may want to pass a URL to this activity (and by then also rename the activity). Or we just start a general "/claim" router which builds different flows depending on user input. Whatever we decide, it will be easy to modify in the future.

Reply:
Yeap was just making sure I understand, this sounds reasonable I think.